src/blog/jwt-identification-and-hot-streams/index.md

```diff
@@ -0,0 +1,173 @@
+# AnyCable PRO:<br>JWT identification and "hot streams"
+
+September 27, 2021
+{date}
+
+When we started collecting applications for the [AnyCable PRO][pro] early access program back in July, we asked respondents to share the way they use _cables_ and which features they would like us to provide out of the box. Analyzing these responses led to the development of two new PRO features, [JWT identification][jwt-id-docs] and [signed streams support][signed-streams-docs]. They might seem unrelated to each other at first glance, but when used together, they have the potential to bring about a huge performance boost. Continue reading to learn more!
+{intro}
+
+<div class="divider"></div>
+
+## Dealing with authentication
+
+Amongst all the asked about features, one of the leading requests was a "token-based authentication". Most Action Cable applications rely on cookies as an authentication mechanism (either directly or via sessions). The main benefit of cookie-based authentication is simplicity–it just works. However, there are some drawbacks:
+
+- Authenticating non-web clients (e.g., mobile apps) is becoming cumbersome.
+- Rails API-only apps usually do not use cookies for web clients authentication as well (they use tokens).
```

Comment:
The phrase "as well" is lost here, so, it doesn't make sense. What would be a synonym? I didn't get it myself. It can't be "too" because it wouldn't fit as a negative sentence.

Reply:
```suggestion
- Rails API-only apps usually do not use cookies for web clients authentication (they use tokens).
```